jsk_rviz_plugins/package.xml

```diff
@@ -49,6 +49,7 @@
   <run_depend>std_msgs</run_depend>
   <run_depend>urdfdom_py</run_depend>
   <run_depend>view_controller_msgs</run_depend>
+  <run_depend>python-scipy</run_depend>
```

Comment:
Thank you for PR.
I am not familiar with python rundepend, but is this no bad effect?

Reply:
I see. Thank you for detailed explanation！